.github/workflows/develop.yml

```diff
@@ -21,6 +21,12 @@ jobs:
         uses: actions/setup-node@v2
 
       - name: Build
+        env:
+          SENTRY_ORG: podverse
+          SENTRY_PROJECT: react
+          SENTRY_AUTH_TOKEN: 4b0697089ef64bf9922dd4697797627f366751ab589b43efb037a41b916cff36
```

Comment:
Should this be a secret?

Reply:
it can't do anything but read some project settings, and shouldn't even be needed anymore, I'll kill it.